style.css

```diff
@@ -0,0 +1,84 @@
+html body {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  background-color: #f7f7f9;
+  height: 718px;
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+}
+
+.wrapper {
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 30px;
+}
+
+.header-container {
+  background-image: url("images/header-shapes-mobile.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+#logo {
+  font-family: Poppins, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+#hello {
+  font-family: Poppins, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #344563;
+  margin-top: 120px;
+}
+
+.about {
+  font-family: Poppins, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 30px;
+}
+
+.social {
+  font-family: Poppins, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-top: 30px;
+}
+
+.social-list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  padding: 0;
+  margin: 30px 0 0 0;
+}
+
+.list-item {
+  list-style: none;
+}
```

Comment:
- [ ] Good job for styling both toolbar and headline section according to Figma design.. Kindly try to  stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- [ ] Both toolbar and headline section must have the margins (top, bottom, left, right) defined in the Template design.
Your Project Look likes this : 
![12](https://user-images.githubusercontent.com/93598189/185573634-525bdcbe-5b30-46c7-8362-93eac807a304.PNG)
![123](https://user-images.githubusercontent.com/93598189/185573636-b8751999-1c20-41a2-a453-79d77586ed85.PNG)


The Figma design Look likes this : 
![Figma design](https://user-images.githubusercontent.com/93598189/185573697-e821ee11-553a-4afb-8264-fe311a941f70.PNG)